test/autoschedulers/mullapudi2016/reorder.cpp

```diff
@@ -81,7 +82,15 @@ double run_test_2(bool auto_schedule) {
         // Provide estimates on the pipeline output
         diff.set_estimates({{0, left_im.width()}, {0, left_im.height()}, {0, 32}, {0, 3}});
         // Auto-schedule the pipeline
-        p.apply_autoscheduler(target, {"Mullapudi2016"});
+        //
+        // Disabling this experimental GPU feature because the autoscheduler correctly
+        // designates dimension 'z' as the stride axis in the GPU grid-stride loop,
+        // which improves thread retention. However, it fails to reorder 'z' inside
+        // the gpu_blocks 'xo' and 'yo', which is required for proper loop nesting
+        // and successful code generation.
+        //
+        // Reference: https://developer.nvidia.com/blog/cuda-pro-tip-write-flexible-kernels-grid-stride-loops/
+        p.apply_autoscheduler(target, get_autoscheduler_params(false));
```

Comment:
I will work on this feature in the next PR.

Reply:
Don't disable the GPU feature here. Simply disable the comparison in the main function  in at line 188 when there is GPU.